src/wmnds/patterns/header/_header.njk

```diff
@@ -16,67 +18,93 @@
-            {{ navItem.name}}
-          </a>
-        </li>
-        {%- endfor %}
-      </ul>
+      {#- If with a title #}
+      {% if params.withTitle -%}
+      <!-- Header title -->
+      <h1 class="wmnds-header__title wmnds-col-1 wmnds-col-sm-auto">
+        Reinstate your Direct Debit<br>(COVID-19)
+      </h1>
+      {%- else %}
+      {#- Else with a navigation -#}
+      <!-- Nav items -->
+      <nav class="wmnds-header__vertical-align wmnds-col-auto">
+        <ul class="wmnds-header__links wmnds-header__main-links">
+          {% for navItem in navItems -%}
+          <li class="wmnds-header__link">
+            <a href="/{{ navItem.name | lower | replace(" ", "-") }}/" title="{{ navItem.name }} page" target="_self" class="wmnds-link" >
+              {{ navItem.name}}
+            </a>
+          </li>
+          {%- endfor %}
+        </ul>
 
-      {%- if params.app %}
-      <!-- Back button -->
-      <ul class="wmnds-header__links wmnds-header__back-btn">
-        <li class="wmnds-header__link">
-          <a
-            href="#"
-            title="Go back to previous page"
-            target="_self"
-            class="wmnds-link"
-            onClick="goBack(event);"
-          >
-            Back
-          </a>
-        </li>
-      </ul>
-      <script>
-        // Function that sends user to previous page they were on,
-        // This can be included in your local javascript file
-        function goBack(e) {
-          e.preventDefault();
-          window.history.back();
-        }
-      </script>
-      <!-- End back button -->
+        {%- if params.app %}
+        <!-- Back button -->
+        <ul class="wmnds-header__links wmnds-header__back-btn">
+          <li class="wmnds-header__link">
+            <a
+              href="#"
+              title="Go back to previous page"
+              target="_self"
+              class="wmnds-link"
+              onClick="goBack(event);"
+            >
+              Back
+            </a>
+          </li>
+        </ul>
+        <script>
+          // Function that sends user to previous page they were on,
+          // This can be included in your local javascript file
+          function goBack(e) {
+            e.preventDefault();
+            window.history.back();
+          }
+        </script>
+        <!-- End back button -->
+        {%- endif %}
+      </nav>
       {%- endif %}
-    </nav>
-    {% endif %}
+    </div>
+  </div>
+
+  {%- if params.banner %}
+  <!-- Phase banner -->
+  <div class="wmnds-container">
+    {{
+      wmndsBanner ({
+        link: 'https://github.com/wmcadigital/wmn-design-system/issues',
+        linkTitle: 'WMN Design System Github'
+      }) | trim | indent(4)
+    -}}
+  </div>
+  {%- endif -%}
+  
+  {%- if params.breadcrumbs and params.breadcrumbs.section != 'Home' %}
+  <!-- Breadcrumbs -->
+  <div class="wmnds-container">
```

Comment:
I think you could wrap these two components with just one "wmnds-container" wrapper.
You could then do an if statement around the master wmnds-container to say "if params.banner OR params.breadcrumbs" and keep your if statements inside to render breadcrumb/banner if available.